docs/spec.md

```diff
@@ -4279,18 +4285,22 @@ ordering of `index_space(result)` and `index_space(operand)`.
 
 | Label | Name      | Type                                  | Constraints |
 |-------|-----------|---------------------------------------|-------------|
-| (I1)  | `operand` | tensor or per-tensor quantized tensor | (C1-C2)     |
+| (I1)  | `operand` | tensor or per-tensor quantized tensor | (C1-C4)     |
```

Comment:
Is there any specific use case for applying to the MHLO.replica_id op with quantization?

Could you explain why we need to add certain restriction (e.g., allowing only per-tensor quantized tensor in the parallelism operator)?

Reply:
That is good point. Thanks!
I will  keep this point in mind while proposing the specification for parallelism operators.